### PR TITLE
Add docs for monitoring V2

### PR DESCRIPTION
**Pull request by aiyengar2**

### CODE REVIEW

**Comment by catherineluse** [comment]
LGTM

File: content/rancher/v2.x/en/monitoring-alerting/2.5.x/rbac/_index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Monitoring creates three `ClusterRoles` and adds Monitoring CRD access to the following default K8s `ClusterRoles`:
+
+| ClusterRole created by chart | Default K8s ClusterRole  | 
+| ------------------------------| ---------------------------|
+| `monitoring-admin` | `admin`|
+| `monitoring-edit`| `edit` |
+| `monitoring-view` | `view `| 
+
+Rancher will continue to use cluster-owner, cluster-member, project-owner, project-member, etc as role names, but will utilize default roles to determine access. For each default K8s `ClusterRole` there are different Istio CRD permissions and K8s actions (Create (C), Get (G), List (L), Update (U), Patch (P), Delete(D), All (*)) that can be performed.
```
Comment (cbron) [inline content/rancher/v2.x/en/monitoring-alerting/2.5.x/rbac/_index.md:42]: > different Istio CRD permissions

File: content/rancher/v2.x/en/monitoring-alerting/2.5.x/rbac/_index.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+| `monitoring-edit`| `edit` |
+| `monitoring-view` | `view `| 
+
+Rancher will continue to use cluster-owner, cluster-member, project-owner, project-member, etc as role names, but will utilize default roles to determine access. For each default K8s `ClusterRole` there are different Istio CRD permissions and K8s actions (Create (C), Get (G), List (L), Update (U), Patch (P), Delete(D), All (*)) that can be performed. 
+
+
+|CRDs                        | Admin | Edit | View | 
+|----------------------------| ------| -----| -----|
+| <ul><li>`monitoring.coreos.com`</li><ul><li>`prometheuses`</li><li>`alertmanagers`</li></ul></ul>| GLW | GLW | GLW|
+| <ul><li>`monitoring.coreos.com`</li><ul><li>`servicemonitors`</li><li>`podmonitors`</li><li>`prometheusrules`</li></ul></ul>| * | * | GLW|
```
Comment (cbron) [inline content/rancher/v2.x/en/monitoring-alerting/2.5.x/rbac/_index.md:48]: Can you add something that corelates which k8s roles tie to which rancher roles. We need to answer the question: _"what access does a project member get by default."_, its hard for customers to figure that out.

File: content/rancher/v2.x/en/monitoring-alerting/2.5.x/rbac/_index.md (hunk 1 above)
Comment (aiyengar2) [reply content/rancher/v2.x/en/monitoring-alerting/2.5.x/rbac/_index.md:42]: resolved in https://github.com/rancher/docs/pull/2719

File: content/rancher/v2.x/en/monitoring-alerting/2.5.x/rbac/_index.md (hunk 2 above)
Comment (aiyengar2) [reply content/rancher/v2.x/en/monitoring-alerting/2.5.x/rbac/_index.md:48]: yes, resolved in https://github.com/rancher/docs/pull/2719

**Pull request merged**